src/routes/mod/[modId]/new-version/+page.svelte

```diff
@@ -121,6 +132,21 @@
           </div>
         </div>
       {/if}
+
+      <div class="p-4">
+        <span>Edit Compatibility Info</span>
+      </div>
+      <div class="card p-4">
+        <EditCompatibilityForm
+          mod={$mod.data.mod}
+          {modId}
```

Comment:
This expects the mods ID, however on the version page, it might also be the mod reference not ID, so you need to pull the ID from the query.